api/public_api_v1.yaml

```diff
@@ -223,18 +222,22 @@ definitions:
     type: string
     description: User's fiscal code.
     pattern: '^[A-Z]{6}[0-9LMNPQRSTUV]{2}[ABCDEHLMPRST][0-9LMNPQRSTUV]{2}[A-Z][0-9LMNPQRSTUV]{3}[A-Z]$'
-  BodyShort:
+  MessageSubject:
     type: string
     description: |-
-      An abstract of the message, in plain text. The content of this field will be delivered to channels that support a limited amount of characters (e.g. SMS, mobile notifications, etc...).
-      The caller should assume that only the content in "body_short" may be delivered so the value of this field MUST contain *all* the required information.
-    minLength: 3
-    maxLength: 100
-  BodyLong:
+      The (optional) subject of the message - note that only some notification
+      channels support the display of a subject. When a subject is not provided,
+      one gets generated from the client attributes.
+    minLength: 10
+    maxLength: 120
+  MessageBodyMarkdown:
     type: string
-    description: 'A long version of the message, in plain text. The content of this field will be delivered to channels that don''t have any limit in terms of content size (e.g. email, etc...). The long version of the message is optional and should be provided only in cases when a longer message (compared to the short version provided in body_short) gives more information to the user. This field should not be a duplicate of body_short.'
+    description: |-
+      The full version of the message, in plain text or Markdown format. The
+      content of this field will be delivered to channels that don't have any
+      limit in terms of content size (e.g. email, etc...).
     minLength: 80
-    maxLength: 8000
+    maxLength: 10000
```

Comment:
can we predict the message size after markdown parsing ? that's because actually we are sending the whole message to queue channels (max 64kb).  is it possible that the generated html has a bigger size so that the payload won't fit in the queue message ?